index.html

```diff
@@ -386,8 +386,7 @@
 <!-- START WELCOME SECTION -->
 
 <section id="welcome-section" class="welcome-section">
-  <h1>Hey I am Sakshi</h1> <br>

```

Comment:
Did you removed the person name ??..
If is it yes , 
**Please add it back.
 Donot remove the person name**

Reply:
Hey I removed the name but i concluded in the website , instead of name I made it more good . The name is in image form that's why I have to remove the name , but its there in form of image. Hope you understand @sudeepshukla338

Reply:
Updated a image , check the website preview image and decide , I can make changes then. @sudeepshukla338